### PR TITLE
fix returning data for virtual accounts.

### PR DESCRIPTION
**Pull request by PawelPawelec-RDX**

### CODE REVIEW

File: src/RadixDlt.NetworkGateway.PostgresIntegration/Services/EntityStateQuerier.cs [hunk 1]
```diff
@@ -729,7 +729,8 @@ private async Task<Entity> GetEntity(GlobalAddress address, GatewayModel.LedgerS
             return entity;
         }
 
-        if (address.ToString()[0] == _ecdsaSecp256k1VirtualAccountAddressPrefix || address.ToString()[0] == _eddsaEd25519VirtualAccountAddressPrefix)
+        var decodedAddress = RadixAddressCodec.Decode(address);
```
Comment (KrzysztofLabus-RDX) [inline src/RadixDlt.NetworkGateway.PostgresIntegration/Services/EntityStateQuerier.cs:732]: How about `firstAddressByte = RadixAddressCodec.Decode(address).Data[0]`?

**Review by KrzysztofLabus-RDX — APPROVED** [review]

File: src/RadixDlt.NetworkGateway.PostgresIntegration/Services/EntityStateQuerier.cs [hunk 2]
```diff
@@ -729,7 +729,8 @@ private async Task<Entity> GetEntity(GlobalAddress address, GatewayModel.LedgerS
...
+        if (decodedAddress.Data[0] == _ecdsaSecp256k1VirtualAccountAddressPrefix || decodedAddress.Data[0] == _eddsaEd25519VirtualAccountAddressPrefix)
         {
             return new VirtualAccountComponentEntity(address);
         }
```
Comment (KrzysztofLabus-RDX) [inline src/RadixDlt.NetworkGateway.PostgresIntegration/Services/EntityStateQuerier.cs:736]: BTW when did we lost support for VirtualIdentities?!

Comment (PawelPawelec-RDX) [reply src/RadixDlt.NetworkGateway.PostgresIntegration/Services/EntityStateQuerier.cs:736]: well, that would be when I introduced that global address type.

Comment (PawelPawelec-RDX) [reply src/RadixDlt.NetworkGateway.PostgresIntegration/Services/EntityStateQuerier.cs:737]: fixed.

**Pull request merged**